README.md

```diff
@@ -1 +1,153 @@
-# Enumerable
\ No newline at end of file
+<a name="readme-top"></a>
+ 
+
+<div align="center"> 
+  <img src="murple_logo.png" alt="logo" width="40"  height="auto" />
+  <br/>  
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Enumerable] <a name="about-project"></a>
+
+>In this project we learn how to use a module inside your class. For this we create a class MyList and a module MyEnumerable
+
+**[your_project__name]**  
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a> 
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://ruby.org/">Ruby</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features" /> 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo" />
+
+- [Live Demo Link]()
+
+## 💻 Getting Started <a name="getting-started" />
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Ruby
+- Git
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+  cd "to where you want to install your project"
+
+ https://github.com/akhror-valiev/decode-a-morse.git
+```
+
+### Install
+
+Install this project with:
+
+```
+ cd my-project
+  gem install
+```
+
+ 
+
+## 👥 Authors <a name="authors" />
+
+👤 **Akhror Valiev**
+
+- GitHub: [GitHub Profile](https://github.com/akhror-valiev)
+- LinkedIn: [LinkedIn Profile](https://www.linkedin.com/in/oshie/)
+
+👤 **Gulalai Ahmadi**
+
+- GitHub: [GitHub Profile](https://github.com/AhmadiGu)
+- LinkedIn: [LinkedIn Profile](https://ahmadigu.github.io/Desktop_Portfolio/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[pending]**
+- [ ] **[pending]**
+- [ ] **[pending]**
```

Comment:
- [ ] Please add features you will or might add to your work in the near future.